src/CodeGenerator/src/Generator/TestGenerator.php

```diff
@@ -261,7 +261,7 @@ private function getInputCode(PhpNamespace $namespace, Shape $shape, bool $inclu
                     return '""';
                 }
 
-                if ('Input' === substr($shape->getName(), -5)) {
+                if ('Input' === substr($shape->getName(), -5) || 'Request' === substr($shape->getName(), -7)) {
```

Comment:
This is statement is still not 100% correct. The generated code is no guarantee that this will be a valid class. 

But Im fine with this for now. It is only test code and it will break loudly when we run the tests.